### PR TITLE
French translation update

### PR DESCRIPTION
**Pull request by nrobadey**

### CODE REVIEW

File: translations/data.fr-FR [hunk 1]
```diff
@@ -94,22 +94,22 @@
       "transparency": "Transparence"
     },
     "locate": {
-      "metersUnit": "",
-      "feetUnit": "",
-      "popup": "",
+      "metersUnit": "mètre",
+      "feetUnit": "pied",
+      "popup": "Dans un préimètre de {distance} {unit} de ce point",
```
Comment (pblottiere) [inline translations/data.fr-FR]:
```suggestion
      "popup": "Dans un périmètre de {distance} {unit} de ce point",
```

Comment (nrobadey) [reply translations/data.fr-FR]: Yes thanks for correcting the typo :)

Comment (pblottiere) [reply translations/data.fr-FR]: Thanks for updating the translation :+1:!

**Comment by manisandro** [comment]
Thanks!

**Pull request merged**